### PR TITLE
RHIDP-2149 chore: Added configuration files for AsciiDoc preview in the IDE

### PR DESCRIPTION
**Pull request by themr0c**
With this update, the documentation author can preview AsciiDoc files in the editor with style.

Supported IDEs: IntelliJ and Visual Studio Code.

Example, in IntelliJ:

Screenshot_20240320_164715

Example, in Visual Studio Code:

Screenshot_20240320_165129

### CODE REVIEW

**Comment by kadel** [comment]
/lgtm
this is great, it makes reviewing in VSCode much easier

File: build/scripts/build.sh [hunk 1]
```diff
@@ -28,14 +28,15 @@ mkdir -p titles-generated/"${BRANCH}";
...
 for t in $(find titles -name master.adoc | sort -uV | grep -E -v "${EXCLUDED_TITLES}"); do
     d=${t%/*}; d=${d/titles/titles-generated\/${BRANCH}}; 
-    CMD="asciidoctor --backend=html5 -o index.html --section-numbers -a toc --failure-level ERROR --trace --warnings --destination-dir $d $t"; 
+    CMD="asciidoctor --backend=html5 -o index.html --section-numbers -a toc --failure-level ERROR --trace --warnings --destination-dir $d -a stylesdir=`pwd`/.asciidoctor -a stylesheet=docs.css $t";
     echo "Building $t into $d ..."; 
     echo "  $CMD"
     $CMD
-    for im in $(grep images/ "$d/index.html" | sed -r -e "s#.+(images/[^\"]+)\".+#\1#"); do 
-        # echo "  Copy $im ..."; 
+    for im in $(grep images/ "$d/index.html" | grep -v 'mask-image' | grep -v 'background' | grep -v 'fa-icons' | grep -v 'jupumbra' | sed -r -e "s#.+(images/[^\"]+)\".+#\1#"); do
```
Comment (nickboldt) [inline build/scripts/build.sh:38]:
```suggestion
    for im in $(grep images/ "$d/index.html" | grep -E -v 'mask-image|background|fa-icons|jupumbra' | sed -r -e "s#.+(images/[^\"]+)\".+#\1#"); do
```

**Review by nickboldt — APPROVED** [review]

**Comment by nickboldt** [comment]
did not test, but looks good; assuming @kadel tested it, let's merge and see what happens! I have already been using asciidoctor previews in vscode but I guess this will apply a more consistent css than the default one

**Comment by nickboldt** [comment]
I get a different markup but the fact that includes are now followed is awesome.

image

**Pull request merged**